ui/redux/reducers/publish.js

```diff
@@ -185,6 +186,9 @@ export const publishReducer = handleActions(
       }
 
       if (!currentUploads[key]) {
+        if (status === 'error' || status === 'conflict') {
+          state.publishError = true;
+        }
         return state;
```

Comment:
:exclamation: We must not update states like this. It just be an immutable update.  `return state` is only used for "no change".

- [ ] Need to do the `return { ...state, publishError: true }` immutable change.
- [ ] Which reducer clears `publishError`?  I think it's `...defaultState` that clears it, but just wanted to double confirm.